node/config.go

```diff
@@ -233,4 +233,9 @@ type Config struct {
 	// ChainDataDir is the root path for per-chain directories where VMs can
 	// write arbitrary data.
 	ChainDataDir string `json:"chainDataDir"`
+
+	// Path to write runtime state to (including PID, API URI, and
+	// bootstrap address). If empty, runtime state will not be
+	// written.
```

Comment:
fwiw I don't think we allow disabling other files by specifying `=""`... But if there is a usecase for it then I think it's fine... `=""` would typically error out otherwise 🤷.